tests/integration-tests/configs/develop.yaml

```diff
@@ -4,6 +4,27 @@ test-suites:
 {% filter indent(2) %}
 {% include 'common/common.yaml' %}
 {% endfilter %}
+  networking:
```

Comment:
let's add the new tests in common.yaml unless we want to exclude them from release tests

Reply:
Done